src/components/Widget/Widget.style.ts

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import fonts from "../../styles/fonts";
+import colors from "../../styles/colors";
+import resetCSS from "../../styles/reset";
+
+export const HintStyled = styled.div`
+  position: absolute;
+  top: 16px;
+  right: 5px;
+`
+
+export const WidgetStyled =  styled.div`
+  ${resetCSS}
+  ${fonts.text};
+  align-items: center;
+  border: 1px solid ${colors.paleBlue2};
+  border-radius: 12px;
+  box-shadow: inset 0 0 10px rgba(46, 125, 175, 0.15);
+  display: flex;
+  gap: 5px;
+  padding: 16px;
+  position: relative;
+  width: 100%;
+
+`
+
+export const WidgetInfoBox = styled.div`
+  margin-right: 10px;
+`
+
+export const WidgetTitle = styled.div`
```

Comment:
a title that is smaller text size than the Info? are these mixed up maybe? Also please consider to use semantic tags like <p> and <h3> etc, divs are non semantic so it makes it hard to less abled users to understand the apps they are trying to use

Reply:
Figma says . Title smaller than Info